lms/djangoapps/teams/views.py

```diff
@@ -877,6 +877,14 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
```

Comment:
Ah I see. I actually just copied this from earlier in this file, worth fixing those too? (around lines 283, 571, 659)